tqdm/_tqdm.py

```diff
@@ -282,6 +285,23 @@ def _decr_instances(cls, instance):
         except KeyError:
             pass
 
+    @classmethod
+    def write(cls, s, file=sys.stderr, end="\n"):
+        """
+        Print a message via tqdm (without overlap with bars)
+        """
+        # Clear all bars
+        for inst in cls._instances:
+            if inst.fp == file:
```

Comment:
Well, no, I don't agree, in fact there is a collusion between `sys.stdout` and `sys.stderr` (they both print in the terminal as the same kind of content), so I will change that so that `.write()` will clean both `sys.stdout` and `sys.stderr`. I guess this is the safest choice for most people, and the less confusing.